sys-apps/plocate/files/plocate.cron

```diff
@@ -1,6 +1,12 @@
 #! /bin/sh
 set -e
 
+# Do not run the cron job if plocate's systemd timer is enabled.
+if which systemctl >/dev/null 2>&1 \
+	&& systemctl is-enabled --quiet plocate-updatedb.timer; then
```

Comment:
There's really no need to call `which` here. The `if` statement could be simplified to this:
```
if systemctl is-enabled --quiet plocate-updatedb.timer >/dev/null 2>&1; then
```
If the `systemctl` command is not found in PATH, the shell will treat it as status 127, and the `exit` command will not be executed.